### PR TITLE
Add DampingForce pybinding

### PR DESCRIPTION
**Pull request by QuantuMope**
This PR simply adds pybinding for `DampingForce`. It also removes unnecessary member attributes in the cpp source. Tested with `spider_example.py` to make sure everything works.

### CODE REVIEW

File: src/rod_mechanics/external_forces/damping_force.cpp [hunk 1]
```diff
@@ -4,41 +4,38 @@
...
 
-DampingForce::DampingForce(const std::shared_ptr<SoftRobots>& m_soft_robots, double m_viscosity)
-    : BaseForce(m_soft_robots), viscosity(m_viscosity) {
+DampingForce::DampingForce(const std::shared_ptr<SoftRobots>& soft_robots, double viscosity)
+    : BaseForce(soft_robots), viscosity(viscosity) {
 }
 
-DampingForce::~DampingForce() {
-}
+DampingForce::~DampingForce() = default;
```
Comment (DezhongT) [inline src/rod_mechanics/external_forces/damping_force.cpp:11]: Just wondering. Why do we need to add = default here?

**Review by DezhongT — COMMENTED** [review]
See the comments before. In addition, I do not find any code changes in spider_example.py for viscosity

File: src/rod_mechanics/external_forces/damping_force.cpp (hunk 1 above)
Comment (QuantuMope) [reply src/rod_mechanics/external_forces/damping_force.cpp:11]: It's essentially equivalent. It's suggested by clang-tidy to use `default` to explicitly show that there is no deconstruction logic.

**Comment by QuantuMope** [comment]
> See the comments before. In addition, I do not find any code changes in spider_example.py for viscosity

Yeah, I didn't want to add damping to the actual example file. I just tested and verified it on my own private code.

**Pull request merged**